.travis.yml

```diff
@@ -10,22 +10,42 @@ sudo: false
 addons:
     apt:
         packages:
-            - graphviz
             - texlive-latex-extra
             - dvipng
 
 env:
     global:
-        - MAIN_CMD='python setup.py'
-        - CONDA_CHANNELS='http://ssb.stsci.edu/conda-dev'
-        - CONDA_DEPENDENCIES='pytest jwst sphinx'
-        - CONDA_JWST_DEPENDENCIES='pytest jwst sphinx'
-        - PIP_DEPENDENCIES=''
+        - CONDA_CHANNELS='http://ssb.stsci.edu/astroconda-dev'
+        - CONDA_DEPENDENCIES="asdf
+                              astropy
+                              crds
+                              dask
+                              drizzle
+                              fitsblender
+                              flake8
+                              gwcs
+                              jsonschema
+                              jplephem
+                              matplotlib
+                              namedlist
+                              numpy
+                              photutils
+                              scipy
+                              six
+                              spherical-geometry
+                              stsci.image
+                              stsci.imagestats
+                              stsci.stimage
+                              stsci.tools
+                              stwcs
```

Comment:
I thought we removed this dependency :confused:

Reply:
Which dependency?

Reply:
Sorry I meant stwcs.

Reply:
It's gone now :)